.github/workflows/build.yml

```
@@ -11,35 +11,30 @@ jobs:
     name: Analyze with SonarCloud
     runs-on: ubuntu-latest
     steps:
+  
       - name: Checkout repository
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)